src/theme/components/link.ts

```diff
@@ -52,6 +52,7 @@ const moreLink = definePartsStyle(
     base: {
       alignItems: "center",
       display: "inline-flex",
+      whiteSpace: "nowrap",
```

Comment:
Why did you add this to just the `moreLink` variant and not to all variants? Icons can be passed into any variant.

Reply:
I wanted to fix the styling on the ones that the DS controls– it felt like if you're adding your own icon, it's your responsibility to style it as necessary?